README.md

```diff
@@ -53,34 +53,42 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
     pip3 install tldr
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+Then you have direct access to less complicated, easier-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`
 
-However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+However, if you just want to browse without an internet connection or installing software,
+checkout the [PDF version](https://tldr.sh/assets/tldr-book.pdf),
+which is always up-to-date with the latest pages and commits.
+
+Also, for browsing without installing a client to your computer,
+see our [web client](https://tldr.ostera.io/)  
```

Comment:
I'd rather not call this "our" client, both because there are more than one ways to consume tldr-pages' content online, and because the tldr-pages project doesn't in fact manage that client. So maybe:

```suggestion
see the web client at <https://tldr.ostera.io>.
```